components/notify/notify.js

```diff
@@ -10,15 +11,30 @@ module.exports = class Notify {
     this.notifyTemplate = options.notifyTemplate;
   }
 
-  send(email) {
+  async sendEmail(templateId, recipient, personalisation) {
     const reference = uuidv4();
 
-    return this.notifyClient.sendEmail(this.notifyTemplate, email.recipient, {
-      personalisation: {
-        'email-subject': email.subject,
-        'email-body': email.body
-      },
-      reference });
+    try {
+      await this.notifyClient.sendEmail(templateId, recipient, {
+        personalisation: personalisation,
+        reference });
+    } catch (error) {
+      console.error('Error sending email:', error.response ? error.response.data : error.message);
+    }
+  }
+
```

Comment:
should we also have a separate sendEmailWithAttachment function?

Reply:
Did some investigation and it is will good but it is out of scope for this bit of work - Please raise it with the project team, thanks